README.md

```diff
@@ -1,3 +1,7 @@
+# &#x1F6A9;The docs have moved!
```

Comment:
Are we able to embed the flag emoji directly? Or is there a benefit to using the `&#x1F6A9;` code?

```suggestion
# 🚩 The docs have moved!
```

Reply:
Oh I did not know we can embed it directly!